lib/Service/BoardService.php

```diff
@@ -641,13 +650,14 @@ public function deleteAcl($id) {
 	 * @throws \OCP\AppFramework\Db\MultipleObjectsReturnedException
 	 * @throws BadRequestException
 	 */
-	public function clone($id, $userId) {
+	public function clone($id, $userId, $withCards = false, $withAssignments = false, $withLabels = false, $withDueDate = false, $moveCardsToLeftStack = false, $restoreArchivedCards = false) {
```

Comment:
We need to add php docs function parameters to the clone function in the `BoardService` class.